### PR TITLE
Fix target for NBD_INFINITYAIO

### PR DESCRIPTION
**Pull request by fabriciopolonio**
Fix target for NBD_INFINITYAIO which currently cannot be built due to missing GYRO and ACC defines.

Issue: https://github.com/betaflight/betaflight/issues/12974

### CODE REVIEW

**Comment by haslinghuis** [comment]
Missing define for OSD and FLASH and maybe BARO

**Comment by fabriciopolonio** [comment]
Added USE_OSD, this FC has no barometer.

File: configs/NBD_INFINITYAIO/config.h [hunk 1]
```diff
@@ -69,6 +74,7 @@
 #define DEFAULT_CURRENT_METER_SCALE 230
 #define BEEPER_INVERTED
 #define FLASH_SPI_INSTANCE SPI1
+#define USE_OSD
```
Comment (haslinghuis) [inline configs/NBD_INFINITYAIO/config.h]: Need to remove. There are no OSD pins defined. Also this is build option
```suggestion

```

Comment (fabriciopolonio) [reply configs/NBD_INFINITYAIO/config.h]: @haslinghuis when you meant OSD you meant the OSD chip?

Comment (fabriciopolonio) [reply configs/NBD_INFINITYAIO/config.h]: This FC has no FLASH, BARO or OSD chips.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**